opencensus/metrics/registry.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+import threading
+
+from opencensus.metrics.export import metric_producer
+
+
+class Registry(metric_producer.MetricProducer):
```

Comment:
Do we want to name this class as `opencensus.metrics.GaugeRegistry` or `opencensus.metrics.gauge.Registry`? `opencensus.metrics.Registry` is a very generic name.

Reply:
That makes sense, `gauge.Registry` works for me.